daemon/src/maker_cfd.rs

```diff
@@ -40,7 +40,11 @@ pub struct NewOrder {
     pub max_quantity: Usd,
 }
 
-pub struct NewTakerOnline {
+pub struct TakerAdded {
+    pub id: TakerId,
+}
+
+pub struct TakerRemoved {
     pub id: TakerId,
 }
```

Comment:
Maybe `TakerConnected` and `TakerDisconnected` would be better names?

Reply:
thanks for the suggestion, they are better. Will change the names & handlers.